plugins/modules/plan_stash.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Aubin Bikouo <abikouo>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = r"""
+---
+module: plan_stash
+version_added: 2.1.0
+short_description: Handle the base64-encoding of a terraform plan file and save it into playbook execution stats.
+description:
```

Comment:
We should add a note here to suggest using `no_log: true`. The plan file can contain unencrypted secrets.